projects/v2/src/app/app.module.ts

```diff
@@ -63,30 +44,12 @@ import { DocsNavModule } from './modules/docs/docs-nav/docs-nav.module';
       disabled: environment.production,
     }),
     NgxsResetPluginModule.forRoot(),
-    OrderModule,
-    FontAwesomeModule,
     MarkdownModule.forRoot(),
-    CompareModule,
-    ControlsModule,
-    DebugLogsModule,
     DoiModule,
```

Comment:
Why is the DoiModule here? Shouldn't it be a part of the root?

Reply:
I have sent them to root module